transcom/mymove
# B 21015 int

## [B-21015](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1005508)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Swagger UI
2. run the v1 prime listmoves api - grab a move uuid
3. run the v3 prime getMoveTaskOrder api
4. verify that county returns for address objects


### Backend



## Screenshots
Before
 
![image](https://github.com/user-attachments/assets/d90d19c7-4b4e-452b-b790-63f4080c460d)

After

 
![image](https://github.com/user-attachments/assets/4c7891de-eee8-461f-b169-3d6db1157053)
